src/test/java/hw4/pages/components/DiffElemPageComponents.java

```diff
@@ -0,0 +1,93 @@
+package hw4.pages.components;
+
+import hw4.data.SelectColor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DiffElemPageComponents {
+
+    public DiffElemPageComponents(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(css = "div.checkbox-row label")
+    private List<WebElement> allCheckboxes;
+
+    @FindBy(css = "ul.panel-body-list.logs")
+    private WebElement logArea;
+
+    @FindBy(css = "ul.panel-body-list.logs li")
+    private List<WebElement> logs;
+
+    @FindBy(css = "div.colors select")
+    private WebElement colorSelect;
+
+    @FindBy(xpath = "//div[3]//label[4]//input[1]")
```

Comment:
If possible, you should use more readable locators (e.g. use ids, static classes  instead of descendant and indices)